app/components/Header.js

```diff
@@ -27,6 +29,10 @@ const styles = {
   icon: {
     composes: ['glyphicon', 'glyphicon-menu-left'],
     fontSize: 32
+  },
+  headerTitle: {
```

Comment:
Since it's already in the `Header` component, we could just call this `title`.
What do you think?